cocos/rendering/custom/builtin-forward-pipeline.ts

```diff
@@ -279,7 +279,7 @@ export class BuiltinForwardPipeline implements PipelineBuilder {
             buildCascadedShadowMapPass(
                 ppl,
                 id,
-                mainLight,
+                mainLight!,
```

Comment:
Fix: 
Error: ../cocos/rendering/custom/builtin-forward-pipeline.ts:282:17 - error TS2345: Argument of type 'DirectionalLight | null' is not assignable to parameter of type 'DirectionalLight'.
  Type 'null' is not assignable to type 'DirectionalLight'.
282                 mainLight,